paasta_tools/tron_tools.py

```diff
@@ -556,68 +552,68 @@ def load_tron_service_config(service, cluster, load_deployments=True, soa_dir=DE
             soa_dir=soa_dir,
         ) for name, job in jobs.items()
     ]
-    return job_configs, extra_config
+    return job_configs
 
 
-def create_complete_config(service, cluster, soa_dir=DEFAULT_SOA_DIR):
-    """Generate a namespace configuration file for Tron, for a service."""
+def create_complete_master_config(cluster, soa_dir=DEFAULT_SOA_DIR):
     system_paasta_config = load_system_paasta_config()
+    config = service_configuration_lib.read_extra_service_information(
+        service_name='tron',
+        extra_info=f'{cluster}/MASTER.yaml',
+        soa_dir=soa_dir,
+    )
+    return format_master_config(
+        config,
+        system_paasta_config.get_volumes(),
+        system_paasta_config.get_dockercfg_location(),
+    )
 
-    job_configs, other_config = load_tron_service_config(
+
+def create_complete_config(service, cluster, soa_dir=DEFAULT_SOA_DIR):
+    """Generate a namespace configuration file for Tron, for a service."""
+    job_configs = load_tron_service_config(
         service=service,
         cluster=cluster,
         load_deployments=True,
         soa_dir=soa_dir,
     )
-
-    if service == MASTER_NAMESPACE:
-        other_config = format_master_config(
-            other_config,
-            system_paasta_config.get_volumes(),
-            system_paasta_config.get_dockercfg_location(),
-        )
-
-    other_config['jobs'] = {
+    preproccessed_config = {}
```

Comment:
nit: don't want to combine this with next line?

Reply:
meh